test/api/requests/organization.test.js

```diff
@@ -154,7 +154,7 @@ describe('Organization API', () => {
       validateAgainstJSONSchema('POST', '/organization/{id}/invite', 400, response.body);
     });
 
-    it('returns the member and the invitation details for cloud.gov origin', async () => {
+    it('returns the member and the invitation details for cloud.gov origin on invite', async () => {
```

Comment:
this is the smallest of nits, but I read this like `invite` is a noun, not a verb. `upon invite` or `upon invitation` or `when invited` or something would help future me know what you meant by returns...details...on invite

Reply:
OK, changed.